modules.json

```diff
@@ -367,4 +367,19 @@
             "ucuser": "Jimbo Wales"
     	}
     }
-]
\ No newline at end of file
+
+    {
+
+        "filename": "get_alllinks_red.py",
+        "docstring": "List all the links and check if the link is a red link or not.",
+        "endpoint": "https://en.wikipedia.org/w/api.php",
+        "params": {
+                "format": "json",
+    		"action": "query",
+    		"generator":"links",
+    		"titles": "Title",
+    		"gpllimit": "max",
```

Comment:
For the purpose of demo app, you could fetch only ~20 pages and use a page that may have many missing links on it already like:
https://en.wikipedia.org/wiki/Wikipedia:Requested_articles/Arts_and_entertainment#Design. You can explore more here: https://en.wikipedia.org/wiki/Wikipedia:Requested_articles.